### ISSUE TITLE
Make the Gruntfile better

### ISSUE THREAD

**Issue by tommaitland** [agency/base#12]

**Comment by ryenjbeatty** [comment]
I think this is still relevant, things like autoprefixing (and refactoring mixins as a result) would be useful, thoughts?

**Comment by tommaitland** [comment]
Yeah I agree, although I think we should break these out into separate
issues. Worth giving auto prefixing a shot.

On Wednesday, 4 February 2015, ryenjbeatty <notifications@github.com> wrote:

> I think this is still relevant, things like autoprefixing (and refactoring
> mixins as a result) would be useful, thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/agency/base/issues/12#issuecomment-72766036>.
>


-- 
Tom Maitland
Senior Web Designer
<http://agency.sc>
agency.sc | Australia's leading creative studio for social change
a: 4/63 Ann Street, Surry Hills NSW 2010
o: +612 9966 4669
m: 0420 372 744
s: tom_maitland_
